cpp/unreal_projects/SpearSim/Python/spear/__init__.py

```diff
@@ -0,0 +1,7 @@
+#
```

Comment:
We shouldn't need this file to be checked in redundantly, assuming that a user follows the `pip install` steps outlined in my previous PR comments.